.github/ISSUE_TEMPLATE/1-bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
```

Comment:
pre label it bug? Will mess a little bit without triage as it will look like someone looked at it and labelled it. But maybe our Zenhub inbox pipeline workflow will cover that

Reply:
We could add a "needs triage" label on all new issues/bugs. would that work with zenhub?

Reply:
zenhub would not be affected by that,  if we view zenhub "Inbox" as needs triage the labeling here would not affect that